install_files/ansible-base/roles/tails-config/files/securedrop_init.py

```diff
@@ -129,6 +129,9 @@
 if is_tails:
```

Comment:
Not strictly related but I think we can remove this call, `gnome-shell-extension-tool` appears to be deprecated.

Reply:
Unless I'm mistaken, I do believe `gnome-extensions` (or its older alias, `gnome-shell-extension-tool`) is still in use, and is necessary here. There are instances, generally outside of Tails installations, where folks will have Shell extensions installed, but not enabled, and without this tool they don't have a way to turn them on and off dynamically while leaving them installed on the system (there are GUI tools like the new Extensions app and GNOME Tweaks that let you do this, but programmatically I believe it's still necessary to use `gnome-extensions`, because even if it's in the right place in the dotfiles, it needs the explicit enable action to launch).

Reply:
AIUI They're two different utilities, not an alias. You do need the gnome-extensions call just below, as that enables your new extension, but the one above is a legacy thing from Tails 4 where it was necessary to reload the desktop icons extension so that the icons would refresh after being enabled. That's the call this refers to.